cpu/cc26x0/cpu.c

```diff
@@ -52,11 +48,3 @@ void cpu_init(void)
     periph_init();
 }
 
```

Comment:
```suggestion
```
[Travis](https://travis-ci.org/github/RIOT-OS/RIOT/builds/741822080) complains about the now trailing whitespace